cmd/gctcli/commands.go

```diff
@@ -4053,10 +4069,11 @@ func getHistoricCandles(c *cli.Context) error {
 				Base:      p.Base.String(),
 				Quote:     p.Quote.String(),
 			},
-			Rangesize:   candleRangeSize,
-			Granularity: candleGranularity,
+			AssetType:      assetType,
+			TimestampStart: start.Unix(),
```

Comment:
Can drop the Timestamp prefix on these and just use start/end like in other commands